quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/CLITestExtension.java

```diff
@@ -249,4 +253,9 @@ private void setProperty(String name, String value) {
     private DistributionTest getDistributionConfig(ExtensionContext context) {
         return context.getTestClass().get().getDeclaredAnnotation(DistributionTest.class);
     }
+
+    private RawDistRootPath getDistPath(){
+        Path distPath = ((RawKeycloakDistribution)dist).getDistPath();
```

Comment:
is it possible to have problems when running the tests multiple times?

Reply:
I didn't observe any while running the testsuite multiple times. Can you elaborate?

Reply:
Just thinking that the distribution folder might be re-used, if you say that it doesn't affect I do trust 👍